lib/utilities/process-template.js

```diff
@@ -0,0 +1,10 @@
+var _template = require('lodash.template');
```

Comment:
@nathanhammond with lazy require I meant moving the above line into the `processTemplate()` function, instead of the more complicated require() above. this solution will work too though. :shrug: